wiki/Registration/de.md

```diff
@@ -1,24 +1,17 @@
----
-outdated: true
-outdated_since: abc38b81368f22f2c02e0feda3b7d5c14fba8ead
----
-
 # Registrierung
 
 *Warnung: Mehr als einen osu!-Account zu besitzen, ist ein Verstoß gegen die allgemeinen [Regeln](/wiki/Rules)!*
 
-1. Nach der [Installation](/wiki/Installation) und dem Starten von osu! fordert dich das Spiel auf, dich einzuloggen oder zu registrieren. Zum Erstellen eines Accounts musst du nur auf `Create an Account` klicken und osu! im Hintergrund offen lassen. Nun öffnet sich die Seite, auf der du einen [Account erstellen](https://osu.ppy.sh/p/register) kannst.
+1. Nach der [Installation](/wiki/Installation) und dem Starten von osu! fordert dich das Spiel auf, dich einzuloggen oder zu registrieren. Zum Erstellen eines Accounts musst du nur auf `Create an account` klicken.
```

Comment:
Maybe should mention here that you that you should not/must not create an account if you already have one. English version does this by saying `Assuming you're here because you need an account...`

Reply:
> Du bist vermutlich hier, da du noch keinen Account hast. Um einen Account zu erstellen, klicke auf `Create an account`.
Is it okay like this (see latest commit)? I didn't want to mention that it is forbidden to create a second account because that is already in the article.

Reply:
That works!